content/dstu2/medication-statement.md

```diff
@@ -26,3 +30,107 @@ Search for MedicationStatements that meet supplied query parameters:
 
 <%= headers 200 %>
 <%= json(:dstu2_medication_statement_bundle) %>
+
+## Create
```

Comment:
We need a lot more documentation around when there needs to be a contained resource and what's required to be filled out when that's there.